C++/Algorithms/Mathematical/SquareRoot.cpp.cpp

```diff
@@ -0,0 +1,51 @@
+#include <iostream>
+using namespace std;
+
+
+//These 2 functions will accept user input and return the resulting Square Root of that user input
+
+
+//Iterative Function
+double sqrtIterative(double x) {
+	double sqrt = x * pow(0.3, log10(x));
+	for (int i = 0; i < 5; i++)
+		sqrt = (sqrt + (x / sqrt)) * 0.5;
+	return sqrt;
+}
+
+
+//Recursive function
+double sqrtRecursive(double num, double prev)
+{
+	double next = (prev + num / prev) / 2;
+	if (fabs(next - prev) < DBL_EPSILON * next)
```

Comment:
Please define the constants.

Reply:
@hemanth-kotagiri as requested I've added comments for the constants.